src/main/java/io/cdap/plugin/google/common/exceptions/InvalidPropertyType.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.google.common.exceptions;
+
+/**
+ * An enum which represents a type of property exception. It is common for both sink and source plugins.
+ */
+public enum InvalidPropertyType {
```

Comment:
why do we need enum for this. 
Can we just use GoogleDriveBaseConfig.AUTH_TYPE as a value instead?

Reply:
Added to GoogleDriveBaseConfig label field with readable name